src/query/storages/fuse-meta/src/caches/cache.rs

```diff
@@ -1,5 +1,3 @@
-// Copyright 2021 Datafuse Labs.
```

Comment:
Why removed?

Reply:
oops, that is a typo, let me add it back